helm/Chart.yaml

```diff
@@ -27,3 +27,12 @@ dependencies:
   - name: nats
     version: "1.3.7"
     repository: "https://nats-io.github.io/k8s/helm/charts/"
+
+annotations:
```

Comment:
Please, revisit the annotations you added with the ones used by kubewarden. See [here](https://github.com/kubewarden/helm-charts/blob/5cae27ee09eac6f9012685f4f4b530cb19b13b78/charts/kubewarden-controller/Chart.yaml#L30-L48)